CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# How to contribute
+We welcome contributions from the community and are pleased to have them.  Please follow this guide when logging issues or making code changes.
+
+## Logging Issues
+All issues should be created using the [new issue form](https://github.com/lob/lob-python/issues).  Clearly describe the issue including steps to reproduce if there are any.  Also, make sure to indicate the earliest version that has the issue being reported.
```

Comment:
Thoughts on having this link to [here](https://github.com/lob/lob-python/issues/new) instead?

Though, I think I may be fore keeping it where it, but also adding a line to make sure they check the closed issues to ensure that it isn't a solved or known problem.